docs/section4/4-19.md

```diff
@@ -0,0 +1,35 @@
+# Airside System - Rule 4-19    
+**Schema Version:** 0.0.13  
+**Mandatory Rule:** True  
+**Rule ID:** 4-19    
+**Rule Description:** Schedules for HVAC fans in the baseline design model that provide outdoor air for ventilation shall be cycled ON and OFF to meet heating and cooling loads during unoccupied hours excluding HVAC systems that meet Table G3.1-4 Schedules per the proposed column exceptions #s 2 and 3.  
+**Rule Assertion:** B-RMR = expected value                                           
```

Comment:
@claperle it would be great to list the entire rule out here. We use these Rule Descriptions to update the Rule Test spreadsheet and want to align them as much as possible. Copied below for your convenience -

2. HVAC fans shall remain on during occupied and unoccupied hours in spaces that have health- and safety mandated
minimum ventilation requirements during unoccupied hours.
3. HVAC fans shall remain on during occupied and unoccupied hours in systems primarily serving computer rooms.